doc/source/dev/conduct/report_handling_manual.rst

```diff
@@ -70,6 +84,49 @@ action deemed necessary.
 How the committee will respond to reports
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+There are two tracks of response; one is specific to clear and severe breaches
+(as defined below).
+
+Clear and severe breach
++++++++++++++++++++++++
+
+We know that it is painfully common for internet communication to start at or
+devolve into obvious and flagrant abuse.  We define a clear and severe breach
+as being: personal threats, violent, sexist or racist language.
```

Comment:
We don't want to be too prescriptive here.

Reply:
This whole section is pretty good, I'm just going to make a minor edit to ensure we don't limit "clear and severe breaches" too much. E.g. deliberately posting other people's personally identifying information is also a clear and severe breach.

Reply:
We do need to be careful not to be too broad or subjective in how these are applied.   We would not want to use this route if the person is being annoying or "difficult". 

How about removing this separate track, and having this as an action that can be applied as a first step after reporting, at the responder's discretion, but where the discretion is bounded by some rules as to what qualifies?    If we do immediately disconnect, then the person has a right to ask for a standard review; we tell them this when we disconnect them, and give them an idea of what arguments they should make in asking for a standard review.

Reply:
That sounds like a good idea.

Reply:
> We do need to be careful not to be too broad or subjective in how these are applied. We would not want to use this route if the person is being annoying or "difficult".

Changed it to "We will deal quickly with clear and severe breaches like personal threats, violent, sexist or racist language.". That's clearly very different from "being annoying".

Reply:
Sure - as long as it's explicit what we are acting on, it's OK.